src/odemis/acq/stream/_helper.py

```diff
@@ -93,6 +93,25 @@ def __init__(self, name, detector, dataflow, emitter, scanner=None, sstage=None,
         #  * Rep set: Rep (as requested) + ROI (current) → PxS (updated)
         #    The repetition is adjusted to fit the hardware limits
 
+        # If "exposureTime" not detvas and detector has VA "exposureTime",
+        # create a new one, which allows image integration.
+        # The range of the new exposureTime VA is longer than the detector is capable of.
+        # Will result in image integration during acquisition.
+        if "detvas" in kwargs.keys() and "exposureTime" not in kwargs["detvas"] and model.hasVA(detector, "exposureTime"):
+
+            self.hwExpTime = self.detector.exposureTime.value
+            # Number of images that need to be acquired for the requested exposure time.
+            # If not integration time, default is 1 image.
+            self.integrationCounts = model.VigilantAttribute(1, readonly=True)
+
+            # increase exposure time range to perform image integration
+            # TODO: for now we specify a max integration time by using a fixed multiple of the exp time
+            # TODO: for now only used in Braunschweig which has max exp time of 10s -> so now 1h
```

Comment:
Indeed, this "6*60" constant is a little arbitrary. Maybe we could just "round" it to 1000, and limit the max time to 1h (either here, or in the gui.data.conf).